lib/instrumentald/server_controller.rb

```diff
@@ -250,5 +229,68 @@ def run
     end
   end
 
+  def run_telegraph
+    instrumentald_pid = Process.pid
+    daemonize_block do
```

Comment:
I think I might just say `daemonize` instead of `daemonize_block` here, since it's taking a block, but neat!